core/src/main/java/io/seata/core/model/BranchStatus.java

```diff
@@ -108,7 +108,7 @@ public int getCode() {
         return code;
     }
 
-    private static final Map<Integer, BranchStatus> MAP = new HashMap<>(values().length);
+    private static final Map<Integer, BranchStatus> MAP = Maps.newHashMapWithExpectedSize(values().length);
```

Comment:
I think that s same and branch status was fixed, it would not let the map resize.

Reply:
This way of writing is bound to expand capacity

Reply:
You re right, thanks.